src/shared/views/BuildingFootprint/FootprintDisplay.js

```diff
@@ -82,7 +140,59 @@ export default class FootprintDisplay extends React.Component<Props> {
         ];
     };
 
-    BBOXToCoords = (bbox: BBOX) => {
+    latLonZoomToPixelCoords = (lonLat: Point, zoom: number): Point => {
+        const [lon, lat] = lonLat;
```

Comment:
maybe we can add a comment here. e.g. this link with further information on what we mean by `PixelCoords`: https://docs.microsoft.com/en-us/bingmaps/articles/bing-maps-tile-system#pixel-coordinates

I think that now we still know, but maybe not in a few weeks, or if someone else will look at the code.

Reply:
good call. Comment added with your link